packages/stargate/src/aminotypes.ts

```diff
@@ -526,6 +526,15 @@ export interface AminoTypesOptions {
 export class AminoTypes {
   private readonly register: Record<string, AminoConverter>;
 
+  // Those message types cannot be signed using the Amino JSON
+  private readonly unsupportedTypes: string[] = [
+    "cosmos.authz.v1beta1.MsgGrant",
+    "cosmos.authz.v1beta1.MsgExec",
+    "cosmos.authz.v1beta1.MsgRevoke",
+    "cosmos.feegrant.v1beta1.MsgGrantAllowance",
+    "cosmos.feegrant.v1beta1.MsgRevokeAllowance",
+  ];
```

Comment:
Can we move those to the config instead of hardcode them?

E.g. if we change

```diff
export class AminoTypes {
-  private readonly register: Record<string, AminoConverter>;
+  private readonly register: Record<string, AminoConverter | "not_implemented_on_chain">;
```

then we can set them explicitly to `"not_implemented_on_chain"`. This allows users
1. to override the value when they do implement it on chain
2. Add their own types with `"not_implemented_on_chain"`

Reply:
Sure, but what config do you mean? Don't we already have such an error handling for unsupported/not implemented amino messages with 

```ts
      throw new Error(
        `Type URL '${typeUrl}' does not exist in the Amino message type register. ` +
          "If you need support for this message type, you can pass in additional entries to the AminoTypes constructor. " +
          "If you think this message type should be included by default, please open an issue at https://github.com/cosmos/cosmjs/issues.",
      );
```
?

Reply:
I mean right now you hardcode `private readonly unsupportedTypes: string[] = [`. This could be made part of the configurable part, i.e.

```ts
function createDefaultTypes(prefix: string): Record<string, AminoConverter | "not_implemented_on_chain"> {
  return {
    // authz

    "/cosmos.authz.v1beta1.MsgGrant": "not_implemented_on_chain", // this setting produces a different error message than a missing typeUrl
    "/cosmos.authz.v1beta1.MsgExec": "not_implemented_on_chain",
    "/cosmos.authz.v1beta1.MsgRevoke": "not_implemented_on_chain",

    // bank

    "/cosmos.bank.v1beta1.MsgSend": {
      aminoType: "cosmos-sdk/MsgSend",
      toAmino: ({ fromAddress, toAddress, amount }: MsgSend): AminoMsgSend["value"] => ({
        from_address: fromAddress,
        to_address: toAddress,
        amount: [...amount],
      }),
      fromAmino: ({ from_address, to_address, amount }: AminoMsgSend["value"]): MsgSend => ({
        fromAddress: from_address,
        toAddress: to_address,
        amount: [...amount],
      }),
    },
```